### PR TITLE
14.1 - Used separate HTML, CSS, and JS files

### PR DESCRIPTION
**Pull request by DChandlerP**
Something about doing it that way just seems cleaner if I'm using Vanilla JS.

### CODE REVIEW

**Comment by DChandlerP** [comment]
This is actually 14.1

File: buildATable/buildATable.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    headers.forEach(function (header) {
+        let headerCell = document.createElement("th");
+        headerCell.textContent = header;
+        headerRow.appendChild(headerCell);
+    });
+    table.appendChild(headerRow);
+
+    anArrayOfMountainObjects.forEach(function (object) {
+        let row = document.createElement("tr");
+        headers.forEach(function (header) {
```
Comment (glenselle) [inline buildATable/buildATable.js:35]: Not a biggie, but you could also have iterated over 
```javascript
Object.keys(object).forEach(function(object) {
```

It's worth noting however, that if you did this you could run the risk of adding a cell for a key on the object that only one of them has. By iterating over `headers`, like you did, it ensures you only add cells that correspond to the headers in your table.

File: Ch 12/language.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        expr.args.push(arg.expr);
+        program = skipSpace(arg.rest);
+        if (program[0] === ",") {
+            program = skipSpace(program.slice(1));
+        } else if (program[0] !== ")") {
+            throw new SyntaxError("Expected ',' or ')'");
+        }
+    }
+    return parseApply(expr, program.slice(1));
+}
```
Comment (glenselle) [inline Ch 12/language.js:67]: Looks like a stray file from another chapter 😉

**Review by glenselle — APPROVED** [review]
Looks good

**Pull request merged**